src/atomic64/fallback.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2014 The Prometheus Authors
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::Atomic;
+use spin::RwLock;
+use std::ops::AddAssign;
+
+pub struct RwlockAtomic<T: AddAssign + Copy> {
```

Comment:
You will find out that it will not compile if you don't add these two bounds :)

`AddAssign` is used here: https://github.com/pingcap/rust-prometheus/pull/157/files#diff-3d526af78389d32fe367216950181fcdR42

notice that `*self.inner.write()` got `T` and `delta` is `T` so that `*self.inner.write() += delta;` required `T: AddAssign`.

`Copy` is used nearby: https://github.com/pingcap/rust-prometheus/pull/157/files#diff-3d526af78389d32fe367216950181fcdR37

```
error[E0507]: cannot move out of borrowed content
  --> src/atomic64/fallback.rs:37:9
   |
37 |         *self.inner.read()
   |         ^^^^^^^^^^^^^^^^^^ cannot move out of borrowed content
```